cmd/connect.go

```diff
@@ -17,15 +17,59 @@ limitations under the License.
 package cmd
 
 import (
-	"encoding/json"
-	"fmt"
+	"context"
 
+	"github.com/meroxa/meroxa-go"
 	"github.com/spf13/cobra"
 )
 
-// ConnectCmd represents the `meroxa connect` command
-func ConnectCmd() *cobra.Command {
-	connectCmd := &cobra.Command{
+type Connect struct {
+	input, config, source, destination string
+}
+
+func (conn *Connect) setFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&conn.source, "from", "", "", "source resource name")
+	cmd.MarkFlagRequired("from")
+	cmd.Flags().StringVarP(&conn.destination, "to", "", "", "destination resource name")
+	cmd.MarkFlagRequired("to")
+	cmd.Flags().StringVarP(&conn.config, "config", "c", "", "connector configuration")
+	cmd.Flags().StringVarP(&conn.input, "input", "", "", "command delimeted list of input streams")
```

Comment:
I think this command could include `--pipeline` as well to make that option available since we're using `create connection` under the hood.